taglib/ape/apetag.cpp

```diff
@@ -53,7 +55,7 @@ namespace
 
   bool isKeyValid(const ByteVector &key)
   {
-    const char *invalidKeys[] = { "ID3", "TAG", "OGGS", "MP+", nullptr };
+    constexpr std::array invalidKeys { "ID3", "TAG", "OGGS", "MP+" };
```

Comment:
This seems like it should/could be static? Not sure why it isn't.

Reply:
Done